app/controllers/case_contacts_controller.rb

```diff
@@ -10,19 +10,30 @@ def index
   # GET /case_contacts/1
   # GET /case_contacts/1.json
   def show
+    @case_contact_number = CasaCase.find(@case_contact.casa_case_id).case_number
   end
 
   # GET /case_contacts/new
   def new
     @case_contact = CaseContact.new
+    @casa_cases = current_user.casa_cases
```

Comment:
I like this! 😃

Reply:
Thank you. This is done because we want to only display the casa cases that belong to the volunteer.